### ISSUE TITLE
Configuration options

### ISSUE THREAD

**Issue by arax** [CESNET/itchy#2]
Can you, please, clarify what the following options (marked with `# <<< ??`) do? Maybe provide a few examples of data placed in those directories by itchy?
```yml
  metadata_dir: /var/spool/onevmcatcher  # <<< ??
  template_dir:                          # <<< ??
  output_dir:                            # <<< ??
  descriptor_dir: /vmcatcher/descriptors # <<< ??
  log_to: stderr
  log_level: error
  ca_path: /etc/grid-security/certificates
  ca_file:
  debug: false
  skip_ca_check: false                   # <<< ??
```

**Comment by arax** [comment]
Why do we have `ca_*` and `skip_ca_*` options in itchy? Is it doing some sort of validation or peer verification?

**Comment by kosoburak** [comment]
Options ca_*, skip_ca_* and template_dir are leftovers after different idea. I forgot to remove them. Sorry for that

metadata_dir is a directory where ITCHY archive stores events from vmcatcher
                      / from where ITCHY process reads stored events for processing
output_dir is a directory where processed images are stored by ITCHY process
descriptor_dir is a directory where ITCHY process stores appliances descriptors (json files for NIFTY)

should I open a pull request with fixed options?

**Comment by arax** [comment]
Yes, please. Thanks!

**Comment by arax** [comment]
Fixed in 7cd77b68d2b34f2199db0038c6b89555fab9ab59
